### PR TITLE
fix(nonci): 'universum run' should still process analyzers correctly

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: universum/nonci.py [hunk 1]
```diff
@@ -16,14 +16,17 @@ def __init__(self, *args, **kwargs):
...
         self.out.log("Cleaning artifacts...")
         self.artifacts.clean_artifacts_silently()
 
         project_configs = self.process_project_configs()
+        afterall_configs = self.code_report_collector.prepare_environment(project_configs)
         self.artifacts.set_and_clean_artifacts(project_configs, ignore_existing_artifacts=True)
 
         self.launch_project()
+        if afterall_configs:
+            self.code_report_collector.report_code_report_results()
```
Comment (i-keliukh) [inline universum/nonci.py:28]: What is the purpose of this line?
There is no vcs and no code review system in nonci mode.

Comment (k-dovgan) [reply universum/nonci.py:28]: If I got it right, this is the function, that transforms intermediate files into final report in artifacts. I might be wrong, though.

Comment (i-keliukh) [reply universum/nonci.py:28]: That is true, but who is supposed to use this report?

Comment (o-andrieiev) [reply universum/nonci.py:28]: test

Comment (o-andrieiev) [reply universum/nonci.py:28]: test

Comment (o-andrieiev) [reply universum/nonci.py:28]: test

Comment (o-andrieiev) [reply universum/nonci.py:28]: test

**Review by i-keliukh — DISMISSED** [review]

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**